modules/ROOT/pages/hardware-and-software-requirements.adoc

```diff
@@ -37,7 +37,7 @@ The Mule runtime engine passed functional testing against the following software
 [%header,cols="15a,85a"]
 |===
 |Software |Version
-| OS | MacOS 10.15, HP-UX 11i V3, AIX 7.2, Windows Server 2019, Windows 10, Solaris 11.3, RHEL 8, Ubuntu Server 20.04
+| OS | MacOS 10.15, HP-UX 11i V3, AIX 7.2, Windows Server 2019, Windows 10, Solaris 11.3, Red Hat Enterprise Linux 8.1 (Ootpa), Ubuntu Server 20.04
```

Comment:
OOTPA is just a nickname for the whole RHEL 8 versions, so let's avoid mentioning it (MacOS and Ubuntu also have them and we are not listing those).